package.json

```diff
@@ -43,7 +43,9 @@
       "__DEV__": true
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "babel-core": "^6.26.3"
```

Comment:
Any reason for this?

Reply:
I was getting an error when running the tests but it's weird, because in the master branch I run the tests once with no problem and then it started complaining. Maybe we can simply remove it and see how it goes.